### ISSUE TITLE
Rewrite Brand Attributes for public consumption

### ISSUE THREAD

**Issue by joshkimmell** [IBM-Watson/design-library#160]
Guidance from Marketing: 
- less aspirational
- more of a statement of guidance
- "not to be construed as "we want Watson to be perceived as trustworthy"

**Comment by bobbinrobyn** [comment]
@joshkimmell: Do you want me to work on this?

**Comment by joshkimmell** [comment]
@bobbinrobyn: If you have time for a rough draft, I'm happy to work on it with you.

**Comment by joshkimmell** [comment]
**Narrowed down to 5 brand statements. Thoughts?**

1. Watson understands natural language and context and can anticipate the user’s needs. 
2. Watson engages the user in dialogue, in context and in any scenario (visual, written, audio).
3. Watson learns both autonomously and via a guide, and improves over time.
4. Watson is capable of machine, as well as human-aided discovery.
5. Watson establishes trust by providing evidence-based, confidence weighted, unbiased and fully transparent outcomes.

**Comment by bobbinrobyn** [comment]
@joshkimmell: I think we probably need more content or this should share a page with something else. I like the direction you're moving in with less text than the original page of brand attributes. But, one thing the original page had going for it was the immediately consumable titles (e.g. "Watson understands me"). These 5 statements feel less immediate to me.

But maybe if you bolded / styled part of them, that would help. Something like this:
**Watson understands** natural language and context and can anticipate the user's needs.
**Watson engages the user** in dialogue, in context and in any scenario (visual, written audio).

**Comment by kathrynmcelroy** [comment]
I agree that this isn't enough content for a whole page, unless we include "Details" for each statement.

Content-wise here's my individual feedback:

1. Watson understands natural language and context and can anticipate the user’s needs.
     - this one is good to go
2. Watson engages the user in dialogue, in context, and in any scenario (visual, written, and audio).
     - couldn't tell if this was a list (which needs an oxford comma), or more like "engages the user in dialogue, within a context and within any scenario"
3. Watson learns both autonomously and via a guide, and improves over time.
     - I think the "via a guide" could be a little clearer. I'm not sure if it's referencing a person or not
4. Watson is capable of machine, as well as human-aided discovery.
      - this is the only one without a power verb, maybe reword to say "Watson discovers insights through machine capabilities as well as human-aid. (needs more work for the latter part...)
5. Watson establishes trust by providing evidence-based, confidence-weighted, unbiased, and fully transparent outcomes.
     - needs another dash and a comma

**Comment by joshkimmell** [comment]
## **Watson understands** natural language and context and can anticipate the user’s needs. 

## **Watson engages the user** in dialogue, in context and in any scenario: visual, written, and even audial. 

## **Watson learns** both autonomously and via a guide, and improves over time. 

## **Watson discovers** insights through machine capabilities as well as human aid.  

## **Watson establishes trust** by providing evidence-based, confidence weighted, unbiased, fully transparent outcomes.

**Comment by bobbinrobyn** [comment]
I second @kathrynmcelroy's thoughts. What does "via a guide" mean? And perhaps *human aid* could say something more like "human expertise" or "human collaboration."

**Comment by joshkimmell** [comment]
human guidance? yeah...

**Comment by kathrynmcelroy** [comment]
**Watson understands** natural language and context and can anticipate the user’s needs.

**Watson engages the user** in dialogue, in context, and in any scenario: visual, written, and audio.

**Watson learns** both autonomously and **via a guide**, and improves over time.

**Watson discovers** insights through machine capabilities as well as human aid.

**Watson establishes trust** by providing evidence-based, confidence-weighted, unbiased, and fully transparent outcomes.


Last thing is changing "via a guide" to something more descriptive/specific.

**Comment by joshkimmell** [comment]
both autonomously and with human expertise...

**Comment by Snugug** [comment]
What's the status on this?

**Comment by joshkimmell** [comment]
Waiting for feedback from Bill in Marketing.

**Comment by poofichu** [comment]
I think this issue needs more labels #kiddingnotkidding

**Comment by bobbinrobyn** [comment]
We're not doing this for the first release. We should move it to 1.1

**Comment by Snugug** [comment]
@bobbinrobyn done
